openfl-workspace/keras_cnn_mnist/src/taskrunner.py

```diff
@@ -3,11 +3,10 @@
 
 """You may copy this file as the starting point of your own model."""
 
-import tensorflow.keras as ke
-from tensorflow.keras import Sequential
-from tensorflow.keras.layers import Conv2D
-from tensorflow.keras.layers import Dense
-from tensorflow.keras.layers import Flatten
+from keras.models import Sequential
```

Comment:
Tensorflow Keras is still evident and we see many examples in the ML community for the same.
Shouldn't we add separate workspace for Keras 3.0 for now and remove this eventually. 

Or we are saying that we are not going to support tensorflow keras anymore.

Reply:
You're right that we should not remove support for tf-keras. We will eventually update the existing tf workspaces to tf-keras and deprecate native tf support since keras has been adopted as the "official" high-level API for tf. 

The goal in this PR is to have an entirely native keras task runner and accompanying workspace since we will eventually want to support jax-keras and torch-keras as well.

Reply:
workspaces for older keras are removed/updated in this PR and  support for tf - keras will be marked deplrecated in https://github.com/securefederatedai/openfl/pull/1204 and eventually removed in future.